### PR TITLE
cs

### PR DESCRIPTION
**Pull request by gauthierm**

### CODE REVIEW

File: BotrAPI.php [hunk 1]
```diff
@@ -1,28 +1,109 @@
...
+
+/**
+ * PHP client library for Bits on the Run System API
  *
  * For the System API documentation see:
- *              http://developer.longtailvideo.com/botr/system-api/
- *-----------------------------------------------------------------------------
+ * http://developer.longtailvideo.com/botr/system-api/
+ *
+ * @category  Services
```
Comment (Qcode) [inline BotrAPI.php:55]: Should these @ tags be repeated twice?

File: BotrAPI.php [hunk 2]
```diff
@@ -97,57 +203,82 @@ private function _args($args)
...
+     * Uploads a file to BotR
+     *
+     * @param array  $upload_link information about the file to upload.
+     * @param string $file_path   path to the file to upload.
+     * @param string $api_format  optional. The requested response format.
+     *                            Defaults to PHP.
+     *
+     * @return array the response object.
+     */
+    public function upload(array $upload_link, $file_path, $api_format = "php")
```
Comment (Qcode) [inline BotrAPI.php:260]: Should the $upload_link still be an empty array or is that safety precaution unnecessary?

File: BotrAPI.php (hunk 1 above)
Comment (gauthierm) [reply BotrAPI.php:55]: Yeah. PEAR standard requires file-level and class-level documentation with those tags.

File: BotrAPI.php (hunk 2 above)
Comment (gauthierm) [reply BotrAPI.php:260]: The original default empty array could never work as the `$file_path`is required. I think it was intended to be type hinting and updated it accordingly.

**Pull request merged**